tests/sentry/web/frontend/test_auth_login.py

```diff
@@ -10,27 +10,38 @@
 from django.urls import reverse
 from django.utils import timezone
 
-from sentry import newsletter, options
+from sentry import newsletter
 from sentry.auth.authenticators import RecoveryCodeInterface
 from sentry.auth.authenticators.totp import TotpInterface
 from sentry.models import AuthProvider, OrganizationMember, User
 from sentry.models.organization import Organization
 from sentry.receivers import create_default_projects
+from sentry.silo import SiloMode
 from sentry.testutils import TestCase
 from sentry.testutils.helpers.features import with_feature
 from sentry.testutils.hybrid_cloud import HybridCloudTestMixin
-from sentry.testutils.silo import control_silo_test
+from sentry.testutils.silo import assume_test_silo_mode, control_silo_test
 from sentry.utils import json
 
 
 # TODO(dcramer): need tests for SSO behavior and single org behavior
-# @control_silo_test(stable=True)
-@control_silo_test
+@control_silo_test(stable=True)
 class AuthLoginTest(TestCase, HybridCloudTestMixin):
     @cached_property
     def path(self):
         return reverse("sentry-login")
 
+    def setUp(self) -> None:
+        from sentry.utils import redis
+
+        # Unfortunately, ratelimiting, which is backed by redis, will fail this test when it runs
+        # twice in silo mode, so we're forced to clear the redis cluster to block the rate limiting.
+        redis.redis_clusters.get("default").flushall(asynchronous=False)
```

Comment:
If we end up needing this in more endpoint tests we can extract to conftest. Adding overhead to all tests just to resolve this problem seems heavy handed.

Reply:
I agree